cmd/soroban-rpc/lib/preflight/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "preflight"
 version = "0.8.0"
 publish = false
-rust-version = "1.69"
```

Comment:
Fyi this `rust-version` field is a minimum version, not a pinned version. The preflight crate will still be compiled with 1.70 if you have 1.70 installed.

Reply:
@leighmcculloch In this PR I have disabled the mechanism that treated this as pinned version. Yes, I'm aware that from rust perspective, this is a min compiler version.